go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/asticode/go-astilectron-bundler
```

Comment:
I'd rather not complexify the PR with adding module support. Can you remove this and make it in another PR?